Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -102,9 +103,7 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
     /// This method will return the remote server list if available, or the local server list if there was a problem with the service call.
     ///
     public func refreshServerList() async throws -> [NetworkProtectionServer] {
-        guard let token = try? tokenStore.fetchToken() else {
-            throw NetworkProtectionError.noAuthTokenFound
-        }
+        let token = try await VPNAuthTokenBuilder.getVPNAuthToken(from: tokenProvider, policy: .localValid)
```

Comment:
Could you wrap this in a `NetworkProtectionError` with underlying error data?

It'll make it much easier to debug issues if we know where this error is coming from.

Reply:
I have modified Networking.OAuthClientError.missingTokens by adding an underlying error and used in here